### PR TITLE
Pin github actions and add dependabot actions ecosystem

### PR DESCRIPTION
**Pull request by MartinHjelmare**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
-  open-pull-requests-limit: 10
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
+  - package-ecosystem: pip
+    directory: "/"
+    schedule:
+      interval: daily
```
Comment (ludeeus) [inline .github/dependabot.yml:11]: Is daily really needed?

```suggestion
      interval: weekly
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
 updates:
-- package-ecosystem: pip
-  directory: "/"
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 10
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
```
Comment (ludeeus) [inline .github/dependabot.yml:6]: Is daily really needed?

```suggestion
      interval: weekly
```

**Review by ludeeus — APPROVED** [review]

**Review by frenck — APPROVED** [review]

File: .github/dependabot.yml (hunk 2 above)
Comment (frenck) [reply .github/dependabot.yml:6]: As per comment on the other PR, the GitHub actions bump does not spam/bump that much.

Comment (MartinHjelmare) [reply .github/dependabot.yml:6]: I'd like to keep it like that for now to not add more load on Monday - Tuesday when the pip bumps come in. If we decide another schedule we can do that later.

Comment (ludeeus) [reply .github/dependabot.yml:6]: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleday

**Pull request merged**